test/index.spec.ts

```diff
@@ -31,16 +31,16 @@ import * as fixturesPromises from './fixtures/index.js';
 const cli = new ESLint();
 
 const configuredRules = {
-  arrayFunc: arrayFunc.rules,
-  eslint: eslintConfigured.rules,
-  eslintComments: eslintComments.rules,
-  functional: functional.rules,
-  import: importConfigured.rules,
-  jest: jestConfigured.rules,
-  simpleImportSort: simpleImportSort.rules,
-  stylistic: stylisticConfigured.rules,
-  typescriptEslint: typescriptEslint.rules,
-  unicorn: unicorn.rules,
+  arrayFunc: arrayFunc[0].rules,
+  eslint: eslintConfigured[0].rules,
+  eslintComments: eslintComments[0].rules,
+  functional: functional[0].rules,
+  import: importConfigured[0].rules,
+  jest: jestConfigured[0].rules,
```

Comment:
nit: I'm not sure myself if it's worth it, but wonder if we should copy-paste existing tests for `jest` rules to have tests for `vitest` rules as well

Reply:
I'm mixed on it too, honestly.  I thought about doing the same but the scope was just getting so big.